cms/templates/webinar_page.html

```diff
@@ -11,9 +11,9 @@
 {% block content %}
 
   {% if page.banner_image %}
-    <div class="resource-banner" style="background-image:url('{% image_version_url page.banner_image "original" %}');background-size:cover">
+    <div class="webinar-resource-banner sub-banner" style="background-image:url('{% image_version_url page.banner_image "fill-1440x200" %}');background-size:cover">
   {% else %}
-    <div class="resource-banner" style="background-image:url('{% static webinar_default_images|random %}');background-size:cover;">
+    <div class="webinar-resource-banner sub-banner">
   {% endif %}
```

Comment:
We would not be needing this if/else if we could return the banner URL from the backend. So this would look something like below without any if else. And we will remove [banner url from CSS below](https://github.com/mitodl/mitxpro/pull/2704/files#diff-d5f636a78a4b410d95575fd299d56a0ffc84c1e3bccd7321f8f3c6c5ffcf06c7R136)
```
<div class="webinar-resource-banner sub-banner" style="background-image:url('{% image_version_url page.banner_image "fill-1440x200" %}');background-size:cover">
```